src/main/java/com/eternalcode/combat/fight/FightManager.java

```diff
@@ -23,10 +33,12 @@ public boolean isInCombat(UUID player) {
     }
 
     public void untag(UUID player) {
+        this.eventCaller.callEvent(new FightUntagEvent(Bukkit.getPlayer(player)));
```

Comment:
Consider an alternative solution - You pass the player's uuid and then get player again, which may not be an efficient solution.
```suggestion
        this.eventCaller.callEvent(new FightUntagEvent(Bukkit.getPlayer(player)));
        
```